### PR TITLE
Add qemu-guest-agent support for v2v on rhel8

### PR DESCRIPTION
**Pull request by xiaodwan**
Add windows and linux guests support for qemu-guest-agent feature

Signed-off-by: Xiaodai Wang <xiaodwan@redhat.com>

### CODE REVIEW

**Comment by xiaodwan** [comment]
depends on https://github.com/avocado-framework/avocado-vt/pull/2575

**Comment by xiaodwan** [comment]
RHEL8
 (1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.OGAC.rhel8.esx_67.rhev.rhv_upload.it_vddk.vpx_uri: PASS (2187.38 s)
 (1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.windows.OGAC.win2019.esx_67.rhev.rhv_upload.it_vddk.vpx_uri: PASS (1121.86 s)
 (1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.windows.OGAC.win2016.esx_67.rhev.rhv_upload.it_vddk.vpx_uri: PASS (2408.45 s)
 (1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.windows.OGAC.win2012.esx_67.rhev.rhv_upload.it_vddk.vpx_uri: PASS (854.83 s)
 (1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.windows.OGAC.win10.esx_67.rhev.rhv_upload.it_vddk.vpx_uri: PASS (2737.25 s)
 (1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.windows.OGAC.win8_1.esx_67.rhev.rhv_upload.it_vddk.vpx_uri: PASS (1832.96 s)
 (1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.windows.OGAC.win8.esx_67.rhev.rhv_upload.it_vddk.vpx_uri: PASS (993.29 s)
  (1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.windows.OGAC.win2012r2.esx_67.rhev.rhv_upload.it_vddk.vpx_uri: PASS (770.07 s)
 (1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.OGAC.rhel6.esx_67.rhev.rhv_upload.it_vddk.vpx_uri: PASS (1339.17 s)
  
RHEL7
(1/1) type_specific.io-github-autotest-libvirt.function_test_esx.positive_test.OGAC.rhel7.esx_67.rhev.rhv_upload.it_vddk.vpx_uri: PASS (521.06 s)

**Comment by xiaodwan** [comment]
@chunfuwen @chloerh @yantian0 Please help to review this PR, thanks.

File: v2v/tests/src/function_test_esx.py [hunk 1]
```diff
@@ -204,7 +213,38 @@ def check_file_architecture(vmcheck):
...
+            logging.debug('Windows guest is rebooting')
+            if vmcheck.session:
+                vmcheck.session.close()
+            vmcheck.create_session()
+            res = utils_misc.wait_for(
+                lambda: re.search(
+                    'running',
+                    vmcheck.get_service_info('qemu-ga'),
+                    re.I),
+                300,
```
Comment (yantian0) [inline v2v/tests/src/function_test_esx.py:241]: Any difference between wait for 600 in the try block with re-create session to wait for 300 in the exception?

Comment (xiaodwan) [reply v2v/tests/src/function_test_esx.py:241]: I don't understand your question. The timeout value is 300s, there is no difference.

Comment (yantian0) [reply v2v/tests/src/function_test_esx.py:241]: Ok, communicated offline.

**Pull request merged**